### PR TITLE
Update logger.conf.gtpl

### PR DESCRIPTION
**Pull request by TECH7Fox**
Fix verbose in `logger.conf`.

image

What do you think @felipecrs? It's from 1 to 3.

### CODE REVIEW

**Comment by TECH7Fox** [comment]
It also seems to be the same with `debug`.

image

**Comment by TECH7Fox** [comment]
Not sure about the log levels, you can change them if you want.

File: asterisk/rootfs/usr/share/tempio/logger.conf.gtpl [hunk 1]
```diff
@@ -14,15 +14,15 @@ console => warning,error
...
-console => verbose,debug,notice,warning,error
+console => verbose(1),debug(1),notice,warning,error
 
 {{- else if eq .log_level "trace" -}}
-console => verbose,debug,notice,warning,error
+console => verbose(2),debug(2),notice,warning,error
 
 {{- else if eq .log_level "all" -}}
-console => dtmf,fax,verbose,debug,notice,warning,error
+console => verbose(3),debug(3),*
```
Comment (felipecrs) [inline asterisk/rootfs/usr/share/tempio/logger.conf.gtpl:26]: Did you test the "*"?

Comment (TECH7Fox) [reply asterisk/rootfs/usr/share/tempio/logger.conf.gtpl:26]: No not yet. Saw it from [here](https://github.com/asterisk/asterisk/blob/master/configs/samples/logger.conf.sample#L157).

**Comment by TECH7Fox** [comment]
Tested, verbose and debug still needed to be set in `asterisk.conf`.

**Pull request merged**